### PR TITLE
dev-util/webstorm (new ebuild)

### PR DESCRIPTION
**Pull request by switch87**
def-util/webstorm: new ebuild

Signed-off-by: Gert Pellin <gert@gepe-biljarts.be>

### CODE REVIEW

**Comment by hasufell** [comment]
@gentoo/java

**Comment by hasufell** [comment]
@chewi @monsieurp

File: dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=virtual/jdk-1.7"
+S=${WORKDIR}/${PN}-${MY_PV}
+
+src_install() {
+	local dir="/opt/${PN}"
+
+	insinto "${dir}"
+	doins -r *
```
Comment (hasufell) [inline dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild]: does `doins -r .` not work? shell expansion is slow

File: dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	fperms 755 ${dir}/bin/{${PN}.sh,fsnotifier{,64}}
```
Comment (hasufell) [inline dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild]: better quote `"${dir}" hier, just for consistencies sake

**Comment by monsieurp** [comment]
I will look at it today.

**Comment by mgorny** [comment]
...said @monsieurp 7 days ago.

**Comment by hasufell** [comment]
:wine_glass:

**Comment by switch87** [comment]
@chewi @monsieurp

**Comment by hasufell** [comment]
I'll merge this in 1-2 days if no one from java herd reviews... @switch87 make sure the license commit message is as follows:
```
licenses: add WebStorm license for dev-util/webstorm
```

**Comment by monsieurp** [comment]
Sorry for not getting back to you ealier. It's really difficult to follow conversations via github due to the amount of noise notifications generate in my inbox. I'm going to merge this PR now.

**Comment by monsieurp** [comment]
>>> Unpacking source...
>>> Unpacking WebStorm-10.0.4.tar.gz to /var/tmp/portage/dev-util/webstorm-10.0.4.141.1550/work
>>> Source unpacked in /var/tmp/portage/dev-util/webstorm-10.0.4.141.1550/work
 * ERROR: dev-util/webstorm-10.0.4.141.1550::gentoo failed (prepare phase):
 *   The source directory '/var/tmp/portage/dev-util/webstorm-10.0.4.141.1550/work/webstorm-141.1550' doesn't exist

**Comment by monsieurp** [comment]
--- a/dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild
+++ b/dev-util/webstorm/webstorm-10.0.4.141.1550.ebuild
@@ -18,7 +18,7 @@ IUSE=""
 KEYWORDS="~amd64 ~x86"

 RDEPEND=">=virtual/jdk-1.7"
-S=${WORKDIR}/${PN}-${MY_PV}
+S="${WORKDIR}/${MY_PN}-${MY_PV}"

:)

**Comment by monsieurp** [comment]
Merged :+1:

**Comment by switch87** [comment]
ok tnx, I was ready with fixing it myself two :+1:

**Pull request closed without merging**